### DISCUSSION TITLE
zh-cn/posts/2024/08/03/20240803-current-website-changes/

### DISCUSSION THREAD

**Discussion by giscus** [CyrusYip/blog-comments#78] [Comments]
# zh-cn/posts/2024/08/03/20240803-current-website-changes/

本文列举了本站于2024年7月28日至2024年08月03日的改动。 清理无用文件 使用我写的 Yue 主题（感觉自己从几乎不折腾博客一下走到折腾博客的尽头了，用自己的主题太爽了，哈哈） 讲 utterances 评论服务改为 giscu

https://cyrusyip.org/zh-cn/posts/2024/08/03/20240803-current-website-changes/

**Comment by hongtaoh** [comment]
真的很强。[我那个主题](https://github.com/hongtaoh/hugo-ht)虽然也是自己一点点磨出来的，但大体的框架还是别人的 (XMin + Lithium)，你这个貌似是自己完全从 0 开始写的。

**Reply by CyrusYip** [reply]
[Yue 主题](https://github.com/CyrusYip/hugo-theme-yue)确实是从头写的。你也可以试试从头写，用 [`hugo new theme`](https://gohugo.io/commands/hugo_new_theme/) 命令就可以创建一个简单的主题，然后慢慢改。[从零构建Hugo主题 - I :: tomo's blog](https://tomo.dev/posts/create-hugo-theme-from-scratch/part-one/) 这个系列的文章也可以参考。
